packages/components/src/theme/ComposerTheme/Components/Skeleton.ts

```diff
@@ -0,0 +1,12 @@
+import { ComponentStyleConfig, cssVar } from '@chakra-ui/react';
+
+export const Skeleton: ComponentStyleConfig = {
+  baseStyle: ({ theme }) => {
+    const $startColor = cssVar('skeleton-start-color');
+    const $endColor = cssVar('skeleton-end-color');
```

Comment:
we can declare this outside of the function, that should provide better performance

```suggestion

const $startColor = cssVar('skeleton-start-color');
const $endColor = cssVar('skeleton-end-color');
    
export const Skeleton: ComponentStyleConfig = {
  baseStyle: ({ theme }) => {
```